javascript/elements/index.js

```diff
@@ -56,6 +56,15 @@ const cachePlaceholders = e => {
 }
 
 const restorePlaceholders = e => {
+  // we have to opt out of this if the request leading to this is a TF request
+  // if the TF request has been promoted to an advance action
+  // (data-turbo-action="advance"), this callback will fire inadvertently
+  // but the whole page will not be exchanged as in a regular TD visit
+  if (sessionStorage.getItem('requested-turbo-frame')) {
+    delete sessionStorage.removeItem('requested-turbo-frame')
```

Comment:
Would it make sense to namespace the session storage key like `futurism-requested-turbo-frame`?

Reply:
yep, but I had second thoughts because [this](https://github.com/stimulusreflex/futurism/pull/181/files#diff-e2d4bb1d32fd47834d1fd7f967e1b5b5cc9d9af5f53b90b2f2a4cd35a15e914eR72) might become brittle 🤔